bsc.bluetcl/targeted/type/polyfield.tcl

```diff
@@ -4,6 +4,7 @@ puts [bpackage load PolyField]
 
 puts {----------}
 
+# Note that this includes defs for generated poly field wrapper structs
```

Comment:
If the wrapper structs no longer appear in this output, then you should delete this comment